src/app/Plans/components/MigrateOrCutoverButton.tsx

```diff
@@ -1,71 +1,58 @@
 import * as React from 'react';
-import { createPortal } from 'react-dom';
 import { useHistory } from 'react-router-dom';
 import { Button, Spinner } from '@patternfly/react-core';
 import spacing from '@patternfly/react-styles/css/utilities/Spacing/spacing';
 import { useCreateMigrationMutation, useSetCutoverMutation } from '@app/queries';
-import { IMigration } from '@app/queries/types/migrations.types';
 import { IPlan } from '@app/queries/types';
 import { PlanActionButtonType } from './PlansTable';
-import { ResolvedQuery, QuerySpinnerMode } from '@app/common/components/ResolvedQuery';
+import MigrateOrCutoverConfirmModal from './MigrateOrCutoverConfirmModal';
 
 interface IMigrateOrCutoverButtonProps {
   plan: IPlan;
   buttonType: PlanActionButtonType;
   isBeingStarted: boolean;
-  errorContainerRef: React.RefObject<HTMLDivElement>;
 }
 
 const MigrateOrCutoverButton: React.FunctionComponent<IMigrateOrCutoverButtonProps> = ({
   plan,
   buttonType,
   isBeingStarted,
-  errorContainerRef,
 }: IMigrateOrCutoverButtonProps) => {
   const history = useHistory();
-  const onMigrationStarted = (migration: IMigration) => {
-    history.push(`/plans/${migration.spec.plan.name}`);
+  const onMigrationStarted = () => {
+    toggleConfirmModal();
+    history.push(`/plans/${plan.metadata.name}`);
   };
   const createMigrationMutation = useCreateMigrationMutation(onMigrationStarted);
-  const setCutoverMutation = useSetCutoverMutation();
```

Comment:
This change makes the page navigate into the migration details when clicking Cutover, which wasn't happening before. But I think it's good to make that consistent with starting the migration. Not sure if you agree @vconzola